pom.xml

```diff
@@ -934,8 +934,8 @@
         <json.wso2.version.range>[3.0.0.wso2v1, 4.0.0)</json.wso2.version.range>
 
         <opensaml.version>3.3.1</opensaml.version>
-        <opensaml2.wso2.version>3.3.1.wso2v1</opensaml2.wso2.version>
-        <opensaml2.wso2.osgi.version.range>[3.3.1,3.4.0)</opensaml2.wso2.osgi.version.range>
```

Comment:
is opensaml2.wso2.osgi.version.range not used anywhere?

Reply:
Fixed with 9e3e81978015b78e7326a9c24958a7ce39be7772